src/main/java/seedu/address/logic/commands/PayCommand.java

```diff
@@ -119,6 +119,8 @@ private static double calculatePaid(Student student, double hour) throws Command
         if (!PaidAmount.isValidPaidAmount(String.format("%.2f", paid))) {
             throw new CommandException(Messages.MESSAGE_LIMIT);
         }
+
+        // Solution below inspired by https://stackoverflow.com/a/28958124
         return BigDecimal.valueOf(paid)
                 .setScale(2, RoundingMode.HALF_UP)
                 .doubleValue();
```

Comment:
Oh, then I think I need to add this into Owe Command also.

Reply:
> Reuse within the team (e.g., reusing code written by a team member) need not be mentioned explicitly. However, you should factor in such reuse when you estimate effort contributed by each team member.

According to reuse policy, I think theres no need to. But if you want to play safe, I think can write in to request to declare also.